sessions/session_store.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019, Google LLC.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/firestore"
+require "rack/session/abstract/id"
+
+module Rack
+  module Session
+    class FirestoreSession < Abstract::Persisted
```

Comment:
This is great... should we add this to `google-cloud-ruby-firestore`?

Reply:
My gut instinct is that it'd have to be a separate gem, and it would need some refining

Reply:
sgtm